jbt-it/neomdb
# 343 client project billing

How to test edit project data:
- Navigate to http://localhost:3000/projekte/1
- Click on "Bearbeiten" at Daten zur Durchführung und Abrechnung
- Check if the nda dialog appears if a new project member is added
- check if the pl warning dialog is added if the members doesnt have the qualification (dummy data not yet connected to the backend)
- check if the qm warning dialog is shown if a new qm is added (dummy data not yet connected to the backend)
- edit the other fields and check if the signature date can be in the future

How to test ProjectBilling:
- go to http://localhost:3000/projekte and than to the tab "Abrechnung"
- open all project billing details
- for project 5 and 9 only the "Dokumenten" should be shown
- for project 9 when clicking on "Abrechnung" a warning should be shown that data is missing
- for project 5 it should be possible to start the billing
- for project 2 also "Informationen zur Projektabrechnung" and "Aufteilung innerhalb des Teams" should be shown
- here you should be able to edit the "BT-Aufteilung" note: it is possible to input more or less BT than the Aufzuteilende BT
